3336/1/top_gems.rb

```diff
@@ -0,0 +1,159 @@
+# rubocop:disable Metrics/AbcSize, Metrics/LineLength, Security/Open
```

Comment:
Need to remove ALL rubocop and reek disables. Only after that I can proceed with review